index.html

```diff
@@ -11,69 +11,338 @@
-               I have a series of experiences in building software for different customers. If you have any project, please contact me.
-           </p>
-           <p>below are my contacts</p>
-        </div>
-        <div class="social-media">
-            <ul class="social-nav">
-                <li><a href="http://linkedin.com/AddisTsega"><i class="fa fa-linkedin"></i></a></li>
-                <li><a href="http://linkedin.com/AdaTsega"><i class="fa fa-twitter"></i></a></li>
-                <li><a href="http://linkedin.com/Aseadd"><i class="fa fa-github"></i></a></li>
-                <li><a href="http://linkedin.com/Addis-Tsega"><i class="fa fa-facebook"></i></a></li>
-            </ul>
-        </div>
-    </section>
-    <section class="placeholder">
-    <div class="recent-works">
-        <div class="project-desc">
-        <div class="work-title">
-            <h3>My Recent Works</h3>
-        </div>
-        <div class="indicator">
-        </div>
-        </div>
-        <div class="card-work-row1" id="card-enable">
-            <div class="placeholder">
-                <div id="project-text1">
-                <div class="project-title1" >
-                    <h3>Multi-post Stories Gain+Glory</h3>
+        <section class="recent-works">
+            <div class="project-text">
+                <div class="projects">
+                    My Recent Works
                 </div>
-                <div class="tags-project1">
-                    <div class="tag1" id="ruby">Ruby on Rails</div>
-                    <div class="tag2" id="css">CSS</div>
-                    <div class="tag3" id="javascript">javaScript</div>
-                    <div class="tag4" id="html">html</div>
+                <div class="indicator">
                 </div>
-                <div class="action-project1">
-                    <button class="btn">id="see-project1">See Projects</button>
+            </div>
+            <div class="grid-work-row1">
+                <div class="card-enable1">
+                    <div class="img-placeholder1">
+                        <div class="project-text">
+                            <div class="project-title">
+                                <h3 class="project-name">Multi-Post Stories Gain+Glory</h3>
+                                <ul class="tags">
+                                        <li class=" tag1"><h6 class="tag-name1">Ruby on rails</h6></li>
+                                       <li class="tag2"><h6 class="tag-name2">CSS </h6></li>
+                                        <li class="tag3"><h6 class="tag-name3">JavaScript</h6></li>
+                                        <li class="tag4"><h6 class="tag-name4">html</h6></li>
+                                </ul>
+                            </div>
+                        </div>
+
+                        <div class="actions">
+                            <div class="btn">
+                                <h6 class="btn-text">See Project</h6>
```

Comment:
To enforce proper use of semantics kindly ensure that **all** _buttons_ are wrapped with the `a` tag, as this serves as link.